viewport-units-buggyfill.js

```diff
@@ -26,7 +26,7 @@
     var initialized = false;
     var options;
     var userAgent = window.navigator.userAgent;
-    var viewportUnitExpression = /([+-]?[0-9.]+)(vh|vw|vmin|vmax)/g;
+    var viewportUnitExpression = /:(\s*)(calc\([\s\S]*)?([+-]?[0-9.]+)(vh|vw|vmin|vmax)/g;
```

Comment:
Since we're trying to avoid base64, wouldn't it make more sense to change the expression so it doesn't match anything starting with `url(`? Right now your pattern would fail to match `transform: translate(2vw, 2vh);`

looks like we've had an attempt at fixing this in the past: https://github.com/rodneyrehm/viewport-units-buggyfill/commit/457e53a5c44d506dfaa6a7e0269c0f8953096e52 - maybe we can fix the base64 problem by fixing `urlExpression = /(https?:)?\/\//` to match `url(` instead?